assistFunctions.c

```diff
@@ -2,8 +2,16 @@
 #include <stdio.h>
 #include "assistFunctions.h"
 
+int min(int N, int* l) {
+  int m = l[0]; // INT8_MAX
+  for (int i = 1; i < N; i++) {
+    if (l[i] < m) m = l[i];
+  }
+  return m;
+}
+
 int max(int N, int* l) {
-  int m = INT8_MIN;
+  int m = 0; // INT8_MIN
```

Comment:
I was getting an error when using INT8_MIN @Ethan-Arrowood, does it matter if we use it?

Reply:
What error? This is as a C default. I'd like to use this, otherwise we need to document what the smallest value in `l` could be and set the initial value of `m` to that.